### ISSUE TITLE
Recommendations for Chat

### ISSUE THREAD

**Issue by TaylorVannatta** [kylecoburn/secret-hitler#33]
It would be nice to filter out the chat to only the actions taken throughout the game, or maybe  a really concise table of who nominated who/what pairs enacted/who investigated who/etc.

Great game though!

**Comment by andy32323** [comment]
You can do a protocol yourself or simply remember these things. This game is so much fun, if you can confuse the participants. Would you want to (partly) take away this fun factor?

**Comment by SanderBuruma** [comment]
@andy32323 yes. the pace of the game is so fast it's impossible to notice and remember enough details without some sort of automated notations.
